.github/workflows/kernel-build.yml

```diff
@@ -140,9 +139,17 @@ jobs:
           max-make-jobs: 32
           llvm-version: ${{ inputs.llvm-version }}
       - name: Tar artifacts
-        working-directory: ${{ env.REPO_ROOT }}
-        run: |
-          bash .github/scripts/tar-artifact.sh ${{ inputs.arch }} ${{ inputs.toolchain_full }}
+        id: tar-artifacts
+        uses: ./tar-artifacts
+        env:
+          ARCHIVE_BPF_SELFTESTS: 'true'
+          ARCHIVE_MAKE_HELPERS: ${{ github.repository != 'kernel-patches/bpf' && 'true' || '' }}
```

Comment:
thought for later: maybe we should move this out of libbpf/ci eventually and let the caller set it through an env var or input (probably preferred)

Reply:
Right, this is why I made them env vars in the first place. They are set in the workflow step right now, I guess you're talking about a case when the workflow itself is called from outside.

When we figure out how to actually use reusable workflows in kernel-patches/vmtest, this might be worth revisiting.